extensions/interactions/base.py

```diff
@@ -143,6 +143,25 @@ def _stats_log_template(self):
             return '{{answer}}'
 
     @property
+    def rules(self):
```

Comment:
Not really sure I agree with that. The plural 'rules' does seem like a list of something to me.